externs.h

```diff
@@ -39,6 +39,7 @@
 #include <pwd.h>
 #include <signal.h>
 #include <stdarg.h>
+#include <stdbool.h>
```

Comment:
With this change you no longer need to explicitly include stdbool.h in database.c

Reply:
ok.